docs/man_pages/screenbuilder/add-about.md

```diff
@@ -33,6 +33,7 @@ Command | Description
 ----------|----------
 [create screenbuilder](../project/creation/create-screenbuilder.html) | Creates a new project for hybrid development with Screen Builder.
 [screenbuilder](screenbuilder.html) | Shows all commands for project development with Screen Builder.
+[upgrade-screenbuilder](upgrade-screenbuilder.html) | Upgrades a project's Screen Builder version
```

Comment:
Also, insert this description in index.md in the SB section as well, just make sure the link is correct there.